portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,243 @@
+                    </a>
+                </div>
+            </div><!--//profile-->              
+        </div><!--//container-->
+        <div class="content">
+            <div class="leftSide">
+                <div class="Aboutme">
+                    <h2>About me</h2>
+                    <p>Hi! I'm Tanya. 
+                    I'm pursuing my Bachelor's in Mathematics and Computing from Indian Institure of Techology, Guwahati. 
+                    Things that interest me the most are music, coding, working out, novels, astrophysics and a starry night sky! 
+                    Also, I love having conversation with all kinds of people and gain experience from them.
+                    <br/>
+                    Recently, I've developed an interest for coding and both the thrill of speed and the patience required for diving deep into problems excite me!
+                    I've also worked on various software projects both during my college and my internships and I've worked hard to learn to write good quality codes.
+                    Feel free to reach out to me anywhere and I'd love to talk to you :)
+                    </p>
+                </div>
+                <div class="projects">
+                    <h2>
+                        Work experience
+                    </h2>
+                    <br />
+                    <br />
+                    <img class="projectLogo" src="../images/uberlogo.jpg">
+                    <div class="projectDesc">
+                        <h3>Uber Assistant Chrome Extension (Ongoing)</h3>
+                        <p>Developed a Node based chrome extension from scratch for assisting Uber employees in their daily work life.
+                            Implemented email addresses based web scraping and RPC Handler to fetch and render employee data on extension.
+                            Worked on fetching the count of documentations contributed by each employee using Lucidworks Fusion framework built on top of Apache Solr.
+                            Implemented document tagging and modified Lucidworks Fusion index and query pipeline for uSearch.
+                        </p>
+                    </div>
+                    <br />
+                    <br />
+                    <img class="projectLogo" src="../images/msLogo.jpg">
+                    <div class="projectDesc">
+                        <h3>Modern Employee Intranet App</h3>
+                        <p>Developed a Modern Employee Intranet app on Microsoft Teams, to host all features and needs
+                            for any organisation on a single platform. Implemented command-based intelligent bots to identify, understand and proactively reply using rich cards, message extension to query through content, and task modules to create a modal popup experience.
+                            The app was created on C# and .NET, used cognitive services like LUIS to develop an NLP based bot.
```

Comment:
Suggestion: When you are talking about not so common acronyms like LUIS, it's good to hyperlink it.

Reply:
Noted! Will link this.